src/GafferCycles/CyclesOptions.cpp

```diff
@@ -123,6 +123,17 @@ CyclesOptions::CyclesOptions( const std::string &name )
 	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_denoise_pass_normal", new IECore::BoolData( true ), false, "useDenoisePassNormal" ) );
 	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:denoiser_prefilter", new IECore::StringData( "accurate" ), false, "denoiserPrefilter" ) );
 
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_guiding", new IECore::BoolData( false ), false, "useGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:deterministic_guiding", new IECore::BoolData( true ), false, "deterministicGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_surface_guiding", new IECore::BoolData( true ), false, "useSurfaceGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:surface_guiding_probability", new IECore::FloatData( 0.5f ), false, "surfaceGuidingProbability" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_volume_guiding", new IECore::BoolData( true ), false, "useVolumeGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:volume_guiding_probability", new IECore::FloatData( 0.5f ), false, "volumeGuidingProbability" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:guiding_training_samples", new IECore::IntData( 128 ), false, "guidingTrainingSamples" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_guiding_direct_light", new IECore::BoolData( true ), false, "useGuidingDirectLight" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_guiding_mis_weights", new IECore::BoolData( true ), false, "useGuidingMisWeights" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:guiding_distribution_type", new IECore::StringData( "PARALLAX_AWARE_VMM" ), false, "guidingDistributionType" ) );
```

Comment:
All these options seem a little overwhelming - I wouldn't know where to begin with things like `surface_guiding_probability`, and the documentation doesn't help. It seems that Blender is exposing a much smaller set of parameters :

![image](https://github.com/GafferHQ/gaffer/assets/1133871/37f836b3-fd3f-4357-9d54-e8f9bf151e28)

Perhaps we should follow suit, especially as it's always possible to throw down a CustomOptions node to set any of the others if you really want to?

Reply:
Blender hides the rest behind two separate checkboxes in the preferences. You have to enable "Developer Extras" and then also enable "Cycles Debug" under "Experimental" to show them, which suggests they're not particularly user-facing...

If we don't remove them outright, then it'd be worth also putting them within an `Advanced` or `Debug` section.

![pathGuidingDebug](https://github.com/GafferHQ/gaffer/assets/50844517/bcabd10f-af88-4aeb-aee7-71812f3bf170)

Reply:
"Debug" definitely implies to me that these aren't things we should expose to users as a matter of course, and CustomOptions is our standard go-to for esoteric stuff like that.